### DISCUSSION TITLE
I have the Github Student Developer Pack but still Codedex unable to verify

### DISCUSSION THREAD

**Discussion by KunalParkhade** [community/community#110417] [GitHub Education] [answered]
### Hi everyone,

I have the access of Github Student Developer Pack but still Codedex unable to verify that as Codedex joins the pack.
![Screenshot 2024-02-28 220845](https://github.com/community/community/assets/113717557/01ae8484-3a34-4084-9201-96ceef19cbb0)

**Comment by samus-aran** [comment, marked as answer]
Hi @KunalParkhade 👋 

Yes, you are confirmed as a verified GitHub Education student. Contact help@codedex.io if the problem persists. 😉  

Visit [Codedex's FAQ](https://www.codedex.io/github-students) for more info:
> If you have questions about this promotion or need assistance with your Codédex account, please send us a message at [help@codedex.io](mailto:help@codedex.io) and our friendly support team will be happy to assist you. :)

**Comment by Yukibei** [comment]
我也是这样的,不知道为什么不能通过

**Comment by Rez-Jr** [comment]
Hello, I am also facing the same issue, and help@codedex.io
 is not responding. I even tried deleting my existing Codédex account, clearing cache and cookies, using different browsers, etc., but the issue still persists. 
<img width="1345" height="917" alt="image" src="https://github.com/user-attachments/assets/cf9c0d63-1f08-49a9-81dd-4b77ed947f7b" />
